app/assets/stylesheets/base.scss

```diff
@@ -1,3 +1,13 @@
 body {
   background-color: white;
 }
+
+.site {
```

Comment:
I feel that adding this styles directly to `body` is ok, no need for a `.site` class.